docs/tutorials/qaoa_with_sampler.ipynb

```diff
@@ -0,0 +1,338 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import rustworkx\n",
+    "from rustworkx.visualization import mpl_draw\n",
+    "\n",
+    "graph = rustworkx.PyGraph.from_adjacency_matrix(w)\n",
+    "layout = rustworkx.random_layout(graph, seed=50)\n",
+    "colors = [\"r\", \"g\", \"b\", \"y\"]\n",
+    "mpl_draw(graph, layout, node_color=colors)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convert graph to a Hamiltonian\n",
+    "\n",
+    "\n",
+    "The graph partition problem can be converted to an Ising Hamiltonian. Qiskit has different capabilities in the Optimization module to perform this. Here, since the goal is to show QAOA with `Sampler`, the module is used without further explanation to create the operator. The paper [Ising formulations of many NP problems](https://arxiv.org/abs/1302.5843) may be of interest if you would like to understand the technique further."
```

Comment:
```suggestion
    "The graph partition problem can be converted to an Ising Hamiltonian. Qiskit has different capabilities in the Optimization module to perform this conversion. Since our goal is to show QAOA with `Sampler`, the module is used without further explanation to create the operator. Review the paper [Ising formulations of many NP problems](https://arxiv.org/abs/1302.5843) to learn more about the technique."
```